server/engine/src/main/java/org/eclipse/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -136,13 +148,20 @@ public class CobolTextDocumentService implements TextDocumentService, ExtendedAp
   private final HoverProvider hoverProvider;
   private final CFASTBuilder cfastBuilder;
   private final ConfigurationService configurationService;
+  private final FileSystemService fileSystemService;
   private DisposableLSPStateService disposableLSPStateService;
   private final CopybookIdentificationService copybookIdentificationService;
   private final CopybookService copybookService;
   private final CopybookReferenceRepo copybookReferenceRepo;
   private final Map<String, Map<String, List<Diagnostic>>> errorsByFileForEachProgram;
   private final SyncProvider syncProvider;
   private final WatcherService watcherService;
+
+  @VisibleForTesting
```

Comment:
It's public, probably it will be visible for testing anyway

Reply:
updated